ddtrace/propagation/http.py

```diff
@@ -60,6 +70,33 @@ def parent_call():
         if span_context.dd_origin is not None:
             headers[HTTP_HEADER_ORIGIN] = str(span_context.dd_origin)
 
+        # Only propagate tags that start with `_dd.p.`
+        tags_to_encode = {}  # type: Dict[str, str]
+        for key, value in span_context._meta.items():
+            # DEV: encoding will fail if the key or value are not `str`
+            key = ensure_str(key)
+            if key.startswith("_dd.p."):
+                tags_to_encode[key] = ensure_str(value)
+
+        if tags_to_encode:
+            encoded_tags = None
+
+            try:
+                encoded_tags = encode_tagset_values(tags_to_encode)
+            except TagsetMaxSizeError as ex:
+                # We hit the max size allowed:
+                #   - add a tag to the context to indicate this happened
+                #   - pass along whatever we were able to encode
+                span_context._meta["_dd.propagation_error"] = "max_size"
+                encoded_tags = ex.current_results
+                log.warning("failed to encode x-datadog-tags", exc_info=True)
+            except TagsetEncodeError:
+                # We hit an encoding error, add a tag to the context to indicate this happened
+                span_context._meta["_dd.propagation_error"] = "encoding_error"
+                log.warning("failed to encode x-datadog-tags", exc_info=True)
```

Comment:
We are changing this, if there is any kind of failure we will not propagate anything